php/test/test_async.php

```diff
@@ -273,27 +279,17 @@ function test_ohlcvs($exchange, $symbol) {
 //-----------------------------------------------------------------------------
 
 function test_symbol($exchange, $symbol, $code) {
-    $method = 'fetchTicker';
-    if ($exchange->has[$method]) {
-        test_ticker($exchange, $symbol);
-    }
-    if ($exchange->id === 'coinmarketcap') {
-        dump(var_export(yield $exchange->fetchGlobal()));
-    } else {
-        yield test_order_book($exchange, $symbol);
-        yield test_trades($exchange, $symbol);
-        yield test_ohlcvs($exchange, $symbol);
-        if ($exchange->check_required_credentials(false)) {
-            if ($exchange->has['signIn']) {
-                $exchange->sign_in();
-            }
-            test_orders($exchange, $symbol);
-            test_closed_orders($exchange, $symbol);
-            test_open_orders($exchange, $symbol);
-            test_transactions($exchange, $code);
-            $balance = yield $exchange->fetch_balance();
-            var_dump($balance);
-        }
+    tester_func('test_ticker', $exchange, $symbol);
```

Comment:
for some reason, it wasn't yielded in last edition, so i've not changed that, probably there was some reason which i am now aware of.